spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1514,15 +1516,15 @@ read.
 fragment to an object.
 
 The following example shows how to define a `StaxEventItemReader` that works with a root
-element named `trade`, a resource of `data/iosample/input/input.xml`, and an unmarshaller
```

Comment:
Figure 1 and section 1.7.1 show the trade example which does not correspond to the content of the `data/iosample/input/input.xml` file. BTW, code examples in the XML section of this chapter refers to the customer credit while the text describing these example talks about the trade example. I made the changes so that both examples and the text describe the trade example (to be consistent with Figure 1 and the xml sample in section 1.7.1). Note that other sections refer to the customer credit sample which is correct there.

Reply:
Good catch. Thanks.